config/locales/en.yml

```diff
@@ -681,6 +681,7 @@ en:
       form_group: "The groups must be from <strong>%{group_min}</strong> to <strong>%{group_max}</strong> people."
       decline_invitation2: "This will decline the invitation you have. Are you sure?"
       your_group: "Your group"
+      your_group_header: "Your Group"
```

Comment:
Can you double-check where `your_group` is used? It might be worth merging these two strings.
Also, when you add the internationalized strings, please add them to the fr and pt files.